scripts/generate-tinysnb.py

```diff
@@ -7,13 +7,19 @@
 # Datasets can only be copied from the root since copy.schema contains relative paths
 os.chdir(KUZU_ROOT)
 
+# Define the build type from input
```

Comment:
You can probably make the script use the relwithdebinfo by default

Reply:
Changed to use relwithdebinfo by default, keep the release option just in case we need in future